doc/en/user/source/community/geofence/configuration.rst

```diff
@@ -0,0 +1,43 @@
+.. _geofence_configuration:
+
+GeoFence Admin GUI
+==================
+
+The GeoFence Admin Page is a component of the GeoServer web interface. You can access it from the GeoServer web interface by clicking the :guilabel:`GeoFence` link, found on the left side of the screen after logging in.
+
+.. figure:: images/configuration.png
+   :align: center
+
+General Settings
+----------------
+Configure the following settings here:
+
+- Geoserver instance name: the name under which this geoserver is known by the geofence server. This useful for when you use an external geofence server with multiple geoserver server.
+
+- GeoServer services URL: this is how geoserver knows how to connect to the external geofence server. When using an internal geofence geoserver, this is not configurable.
+
```

Comment:
The "internal geofence geoserver" probably should be "internal geofence server" or "internal geofrence service". Maybe could use an example here, or a reference to some other docs, for the URL.